cli/ceph/mgr.py

```diff
@@ -24,7 +24,7 @@ def module(self, action, module=None, force=False):
         if force:
             cmd += " --force"
 
-        return self.execute(sudo=True, long_running=True, cmd=cmd)
```

Comment:
Does this impact existing code which is using this method ? instead of removing `long_running`, use `**kw` to pass same parameter.

Reply:
Yes Vaibhav
With parameter "long_running=True" we are getting output in binary (0 or 1) not exact output.
For negative testing to verify error output we need exact output.

Reply:
in some cases we might need to pass this parameter. Instead of removing `long_running` completely, use `**kw` to pass `exec method`

Reply:
Updated